documentation.yaml

```diff
@@ -0,0 +1,66 @@
+apiDoc:
+  title: wix-ui-tpa
+  description: Our official library of React components that are designed to be used in Wix TPA's.
+  category: fe-guild
+  status: ga
+  gitUrl: js
+  markdownOnlyAutoUpdate: true
+  docs:
+    - file: README.md
+      title: Getting Started
+    - file: docs/USAGE.md
+      title: Usage
+    - file: Contribution/docs/CONTRIBUTION.md
+      title: How to Contribute?
+    - file: docs/COMPONENT_GUIDELINES.md
+      title: Contribution/Guidelines for creating a `wix-ui-tpa` component
```

Comment:
Short titles, this is the left side menu:
`Contribution/Guidelines for creating a wix-ui-tpa component` => `Contribution/Guidelines`